ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicySection.tsx

```diff
@@ -92,6 +118,16 @@ function PolicySection({ sectionIndex, descriptors, readOnly = false }: PolicySe
                             hasNoOffset: true,
                             className: undefined,
                         },
+                        selectableActions: {
+                            selectableActionId: `policy-section-${sectionIndex}`,
+                            selectableActionAriaLabelledby: `Policy section ${sectionIndex + 1}: ${sectionName}`,
```

Comment:
Should this be `selectableActionAriaLabel` instead?

I think the `selectableActionAriaLabelledby` property is meant to reference an element used for the aria-label.